upf/upf_cli.c

```diff
@@ -63,8 +63,6 @@ upf_pfcp_endpoint_ip_add_del_command_fn (vlib_main_t * vm,
     {
```

Comment:
Consider using a more descriptive commit message to capture the essence of the changes.